Jakefile.js

```diff
@@ -288,6 +288,9 @@ task('serve', ['default'], { async: true }, function () {
     else if (process.env.packaged === 'true') {
         cmdArg = '-pkg'
     }
+    if (process.env.browser) {
```

Comment:
Those aren't actual environement variables, it's just how jake does argument parsing. See here: http://www.jakejs.com/docs#tasks_passing_parameters_to_jake